com.unity.visualeffectgraph/Editor/ShaderGraph/VFXShaderGraphParticleOutput.cs

```diff
@@ -64,16 +64,16 @@ public override void DisplayWarnings()
                 var shaderGraphParticleOutput = (VFXShaderGraphParticleOutput)target;
                 var shaderGraph = shaderGraphParticleOutput.GetOrRefreshShaderGraphObject();
                 var materialShadowOverride = VFXLibrary.currentSRPBinder.TryGetCastShadowFromMaterial(shaderGraph, shaderGraphParticleOutput.materialSettings, out var castShadow);
-                var materialQueueOffsetOverride = VFXLibrary.currentSRPBinder.TryGetQueueOffset(shaderGraph, shaderGraphParticleOutput.materialSettings, out var queueOffset) && shaderGraphParticleOutput.subOutput.supportsMaterialOffset;
+                var materialSortingPriorityOverride = VFXLibrary.currentSRPBinder.TryGetQueueOffset(shaderGraph, shaderGraphParticleOutput.materialSettings, out var queueOffset) && shaderGraphParticleOutput.subOutput.supportsSortingPriority;
 
                 // Indicate material override from shaderGraph which is hiding output properties.
-                if (materialShadowOverride || materialQueueOffsetOverride)
+                if (materialShadowOverride || materialSortingPriorityOverride)
                 {
                     var msg = "The ShaderGraph material is overriding some settings:";
                     if (materialShadowOverride)
-                        msg += string.Format("\n - castShadow = {0}", castShadow ? "true" : "false");
-                    if (materialQueueOffsetOverride)
-                        msg += string.Format("\n - materialOffset = {0}", queueOffset);
+                        msg += string.Format("\n - Cast Shadow = {0}", castShadow ? "true" : "false");
+                    if (materialSortingPriorityOverride)
+                        msg += string.Format("\n - Sorting Priority = {0}", queueOffset);
```

Comment:
Consider using an `StringBuilder`.

```
StringBuilder stringBuilder = new StringBuilder("The ShaderGraph material is overriding some settings:");
if (materialShadowOverride)
  stringBuilder.AppendLine( "- Cast Shadows: true");
else
stringBuilder.AppendLine( "- Cast Shadows: false");
....
EditorGUILayout.HelpBox(stringBuilder.ToString(), MessageType.Info);
```

Reply:
Fix at d1ea7f9dc14cc507e849e2df4f0ccea0b39493e8 and tested locally
![_string_builder](https://user-images.githubusercontent.com/22494706/135422812-27831cf8-43f0-407f-a92f-12418bde0031.gif)